src/internal.c

```diff
@@ -4069,9 +4076,12 @@ int SetSSL_CTX(WOLFSSL* ssl, WOLFSSL_CTX* ctx, int writeDup)
     #endif
 #endif
 
+    ssl->CBIORecv = ctx->CBIORecv;
```

Comment:
This was added because of wolfSSL_set_bio and compatibility function SSL_set_bio. The function sets a BIO to be used as an IO callback. It expects to be able to adjust the BIO used by an individual WOLFSSL structure without changing the WOLFSSL_CTX IO callback.